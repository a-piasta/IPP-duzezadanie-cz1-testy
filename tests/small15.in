repairRoad;ovfzpnqgruyyarjilsmyizycgpuxp;rhzvirducgiyibqxauojlewfrxa;-3
getRouteDescription;252
extendRoute;50;ovfzpnqgruyyarjilsmyizycgpuxp
addRoad;yvfpdgknyojcw;pebsfgqjfuhenolztepqpxj;27;6
newRoute;562;hkdtquiirgpjgiui;udazcpovxhpyfrpqbci
addRoad;ovfzpnqgruyyarjilsmyizycgpuxp;aqgkldnwbrpnxeqjpqnukyvuzt;0;-9
removeRoad;pebsfgqjfuhenolztepqpxj;yvfpdgknyojcw
extendRoute;562;dtpbxvsruhwonxppofmmmbktvbjwd
addRoad;mgmbjjycculbiyyzqme;ovfzpnqgruyyarjilsmyizycgpuxp;23;2
removeRoad;ovfzpnqgruyyarjilsmyizycgpuxp;aqgkldnwbrpnxeqjpqnukyvuzt
getRouteDescription;562
removeRoad;mgmbjjycculbiyyzqme;ovfzpnqgruyyarjilsmyizycgpuxp
newRoute;677;hkdtquiirgpjgiui;aqgkldnwbrpnxeqjpqnukyvuzt
repairRoad;hkdtquiirgpjgiui;aqgkldnwbrpnxeqjpqnukyvuzt;6
addRoad;pebsfgqjfuhenolztepqpxj;udazcpovxhpyfrpqbci;22;-1
repairRoad;aqgkldnwbrpnxeqjpqnukyvuzt;pebsfgqjfuhenolztepqpxj;13
getRouteDescription;562
removeRoad;pebsfgqjfuhenolztepqpxj;udazcpovxhpyfrpqbci
removeRoad;udazcpovxhpyfrpqbci;pebsfgqjfuhenolztepqpxj
getRouteDescription;677
repairRoad;ovfzpnqgruyyarjilsmyizycgpuxp;dtpbxvsruhwonxppofmmmbktvbjwd;12
newRoute;659;aqgkldnwbrpnxeqjpqnukyvuzt;hkdtquiirgpjgiui
addRoad;dtpbxvsruhwonxppofmmmbktvbjwd;ppqxvfqatiahvukbbdycxcyws;19;-6
removeRoad;ppqxvfqatiahvukbbdycxcyws;dtpbxvsruhwonxppofmmmbktvbjwd
removeRoad;ovfzpnqgruyyarjilsmyizycgpuxp;mgmbjjycculbiyyzqme
extendRoute;562;yvfpdgknyojcw
removeRoad;dtpbxvsruhwonxppofmmmbktvbjwd;ppqxvfqatiahvukbbdycxcyws
addRoad;aqgkldnwbrpnxeqjpqnukyvuzt;aqgkldnwbrpnxeqjpqnukyvuzt;8;-10
extendRoute;659;ovfzpnqgruyyarjilsmyizycgpuxp
newRoute;790;pebsfgqjfuhenolztepqpxj;ovfzpnqgruyyarjilsmyizycgpuxp
removeRoad;ovfzpnqgruyyarjilsmyizycgpuxp;aqgkldnwbrpnxeqjpqnukyvuzt
getRouteDescription;391
newRoute;523;hkdtquiirgpjgiui;rhzvirducgiyibqxauojlewfrxa
newRoute;393;pebsfgqjfuhenolztepqpxj;ovfzpnqgruyyarjilsmyizycgpuxp
removeRoad;pebsfgqjfuhenolztepqpxj;udazcpovxhpyfrpqbci
addRoad;dtpbxvsruhwonxppofmmmbktvbjwd;dtpbxvsruhwonxppofmmmbktvbjwd;26;14
newRoute;139;yvfpdgknyojcw;aqgkldnwbrpnxeqjpqnukyvuzt
newRoute;408;dtpbxvsruhwonxppofmmmbktvbjwd;hkdtquiirgpjgiui
getRouteDescription;139
repairRoad;aqgkldnwbrpnxeqjpqnukyvuzt;udazcpovxhpyfrpqbci;10
getRouteDescription;523
getRouteDescription;659
newRoute;930;ovfzpnqgruyyarjilsmyizycgpuxp;rhzvirducgiyibqxauojlewfrxa
repairRoad;aqgkldnwbrpnxeqjpqnukyvuzt;hkdtquiirgpjgiui;11
extendRoute;408;aqgkldnwbrpnxeqjpqnukyvuzt
repairRoad;rhzvirducgiyibqxauojlewfrxa;yvfpdgknyojcw;-3
newRoute;541;hkdtquiirgpjgiui;pebsfgqjfuhenolztepqpxj
getRouteDescription;677
getRouteDescription;139
newRoute;554;udazcpovxhpyfrpqbci;pebsfgqjfuhenolztepqpxj
removeRoad;pebsfgqjfuhenolztepqpxj;yvfpdgknyojcw
getRouteDescription;811
addRoad;udazcpovxhpyfrpqbci;ppqxvfqatiahvukbbdycxcyws;10;-6
getRouteDescription;930
repairRoad;mgmbjjycculbiyyzqme;dtpbxvsruhwonxppofmmmbktvbjwd;4
extendRoute;139;yvfpdgknyojcw
newRoute;219;mgmbjjycculbiyyzqme;dtpbxvsruhwonxppofmmmbktvbjwd
addRoad;hkdtquiirgpjgiui;dtpbxvsruhwonxppofmmmbktvbjwd;6;-6
