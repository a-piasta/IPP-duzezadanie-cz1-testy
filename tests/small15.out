1: NIE
2: 
3: NIE
4: TAK
5: NIE
6: NIE
7: TAK
8: NIE
9: TAK
10: NIE
11: 
12: TAK
13: NIE
14: NIE
15: TAK
16: NIE
17: 
18: TAK
19: NIE
20: 
21: NIE
22: NIE
23: TAK
24: TAK
25: NIE
26: NIE
27: NIE
28: NIE
29: NIE
30: NIE
31: NIE
32: 
33: NIE
34: NIE
35: NIE
36: NIE
37: NIE
38: NIE
39: 
40: NIE
41: 
42: 
43: NIE
44: NIE
45: NIE
46: NIE
47: NIE
48: 
49: 
50: NIE
51: NIE
52: 
53: TAK
54: 
55: NIE
56: NIE
57: NIE
58: TAK
